### ISSUE TITLE
The "Working..." loading image still keeps running when an error happens

### ISSUE THREAD

**Issue by gastaldi** [openshiftio/launchpad-frontend#67]
![screenshot from 2017-04-12 15-56-20](https://cloud.githubusercontent.com/assets/54133/24974493/b68ee75a-1f98-11e7-8858-a986281c0389.png)

The log says: 
```
Caused by: io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: POST at: https://openshift.default.svc.cluster.local/oapi/v1/projectrequests. Message: project "demo" already exists. Received status: Status(apiVersion=v1, code=409, details=StatusDetails(causes=[], group=null, kind=project, name=demo, retryAfterSeconds=null, additionalProperties={}), kind=Status, message=project "demo" already exists, metadata=ListMeta(resourceVersion=null, selfLink=null, additionalProperties={}), reason=AlreadyExists, status=Failure, additionalProperties={}).
```

It would be nice to show that error in the UI too

**Comment by gastaldi** [comment]
This is still happening in the latest frontend

**Comment by edewit** [comment]
fixed in 66bd953dee4bdebd6568140df1bd452be9d995d4
